tests/phpunit/unit/tests/Shopping/Mock_Product.php

```diff
@@ -0,0 +1,74 @@
+<?php
```

Comment:
This file should maybe live in here - https://github.com/GoogleForCreators/web-stories-wp/tree/main/tests/phpunit/integration/includes

Reply:
Thinking it should stay (at least for now) as the other location is integration Tests